superset/sql_lab.py

```diff
@@ -528,6 +528,8 @@ def execute_sql_statements(  # pylint: disable=too-many-arguments, too-many-loca
 
     if store_results and results_backend:
         key = str(uuid.uuid4())
+        payload["query"]["resultsKey"] = key
```

Comment:
I think instead of adding this line here, it would be more understandable to others to add this into line# 554. Then, others would understand easily that we need to add results_key into payload when we update the db.

Reply:
it's not the same, the key is scoped to this code block